app/Renderers/Layouts/Campaign/CampaignExport.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+namespace App\Renderers\Layouts\Campaign;
+
+use App\Renderers\Layouts\Layout;
+use Illuminate\Support\Facades\Storage;
+
+class CampaignExport extends Layout
+{
+    /**
+     * Available columns
+     * @return array[]
+     */
+    public function columns(): array
+    {
+        $columns = [
+            'created_by' => [
+                'key' => 'user.name',
+                'label' => 'campaigns.exports.created_by',
+                'render' => function ($model) {
+                    $html = '<a class="block break-all truncate" href="' . route('users.profile', [$model->user]) . '" target="_blank">' . $model->user->name . '</a>';
+                    return $html;
+                },
+            ],
+            'size' => [
+                'label' => 'campaigns.exports.size',
+                'render' => function ($model) {
+                    if ($model->size == 0) {
+                        return '<1 MB';
+                    }
+                    return $model->size . ' MB';
```

Comment:
Hopefully it never happens, but adding `number_format()` would help here in case there is ever a >1000MB export.